test/runtests.jl

```diff
@@ -18,6 +18,7 @@ Random.seed!(1)
     @show_requirements solve(solver, mdp)
 
     r = test_solver(solver, mdp, max_steps = 100)
+    @show r
```

Comment:
@NeroBlackstone it's generally better to keep tests as terse as possible (don't print anything unnecessary) unless there is a failure. The @show_requirements is printed by default because it is not trivial to write a test that verifies it. So I will remove the @show r lines